### PR TITLE
Updating of 10 Models & Uploading of relevant textures

### PR DESCRIPTION
**Pull request by PrimeOsiris**
An update on the 10 models I created last Trimester after new experience with unity and Dennis Yu brought issues with the models to my attention on the 12th. The issues were overlapping and disappearing geometry and the textures were not applied to the FBX files. I have updated the models geometry and textures, along with adding the textures to the textures folder and making the path between them relative so that the textures should stick.

### CODE REVIEW

**Comment by DennisPYu** [comment]
The building scale needs to be fixed. Unnecessary object needs to be removed.

**Comment by DennisPYu** [comment]
The building size is still too big. Import the bicycle model in the blender too for comparison.

File: Assets/Models/Buildings/3StoryHome.fbx
Comment (DennisPYu) [inline Assets/Models/Buildings/3StoryHome.fbx:1]: Something wrong that makes
Snipaste_2024-07-22_20-59-18
 part of right side building is tranparent.

Comment (PrimeOsiris) [reply Assets/Models/Buildings/3StoryHome.fbx:1]: Is it just that model I missed? Is the same issue on any other model?

**Review by DennisPYu — COMMENTED** [review]
Only one building left for change. The rest is good.

File: Assets/Models/Buildings/3StoryHome.fbx
Comment (DennisPYu) [reply Assets/Models/Buildings/3StoryHome.fbx:1]: This is only one.

Comment (DennisPYu) [reply Assets/Models/Buildings/3StoryHome.fbx:1]: > Is it just that model I missed? Is the same issue on any other model?
This is only one.

**Review by DennisPYu — APPROVED** [review]

**Pull request merged**